python/paddle/trainer_config_helpers/optimizers.py

```diff
@@ -365,7 +365,8 @@ def settings(batch_size,
              regularization=None,
              is_async=False,
              model_average=None,
-             gradient_clipping_threshold=None):
+             gradient_clipping_threshold=None,
```

Comment:
Seems these changes broke the CI unit tests

Reply:
Done.